src/lsqecc/patches/patches.py

```diff
@@ -49,20 +49,37 @@ def get_graph_edge(edge: Edge) -> Optional[Tuple[Tuple[int, int], Tuple[int, int
 
 
 class EdgeType(Enum):
+    """Enum for representing what an Edge looks like and what type of operation it encodes.
+    When representing Pauli operators, an Edge is (solid or dashed) and (stitched or unstitched).
```

Comment:
Stiched just means that the stabilizers are active on the boundary

Reply:
Yeah leaving the word stitched un-documented might be confusing